python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -48,6 +48,7 @@ def start_ganga(gangadir_for_test, extra_opts=[]):
         ('Configuration', 'gangadir', gangadir_for_test),
         ('Configuration', 'user', 'testframework'),
         ('Configuration', 'repositorytype', 'LocalXML'),
+        ('Configuration', 'UsageMonitoringMSG', False),  # Turn off spyware
```

Comment:
One of the reasons that I turned off the spyware for the tests is that it will make any statistics we gather about user version, backends, applications etc. worthless as it will be flooded by test run data. I think if we want to test that stomp can be submitted in parallel then we should have a test which tests that rather than relying on an unrelated test to do so.